src/engine/level_script.c

```diff
@@ -700,7 +700,25 @@ static void level_cmd_set_mario_start_pos(void) {
 #else
     vec3s_copy(gMarioSpawnInfo->startPos, CMD_GET(Vec3s, 6));
 #endif
-    vec3s_set(gMarioSpawnInfo->startAngle, 0, CMD_GET(s16, 4) * 0x8000 / 180, 0);
+    vec3s_set(gMarioSpawnInfo->startAngle, 0x0, degrees_to_angle(CMD_GET(s16, 4)), 0x0);
+
+#ifdef STARTUP_SPAWN_LEVEL
+    static u8 firstWarp = TRUE;
```

Comment:
Nitpick, but I'm personally not a huge fan of using static function declarations. I'd prefer it to be initialized outside of this function, though I'll let it slide here if you disagree.